modules/packages/LinearAlgebra.chpl

```diff
@@ -1091,7 +1091,44 @@ proc cholesky(A: [] ?t, lower = true)
 }
 
 
-// TODO: add example usage to docs
+/* Find the eigenvalues of matrix ``A``. ``A`` must be square.
+
+    .. note::
+
+      This procedure depends on the :mod:`LAPACK` module, and will generate a
+      compiler error if ``lapackImpl`` is ``none``.
+
+*/
+proc eigvals(A: [] ?t) where isRealType(t) && A.domain.rank == 2 && usingLAPACK {
+  return eig(A, left=false, right=false);
+}
+
+/* To be removed after 1.19.0 */
+pragma "no doc"
+proc eigvals(A: [] ?t, param right) where isRealType(t) && A.domain.rank == 2 && usingLAPACK {
```

Comment:
This probably doesn't matter since these are deprecated anyway, but this lost the ability to call `eigvals` without specifying a name for a second argument like: `eigvals(A, true)` to get the left eigenvectors.